src/main/java/com/gtnewhorizons/angelica/transform/compat/extrautils/RenderBlockColorTransformer.java

```diff
@@ -0,0 +1,39 @@
+package com.gtnewhorizons.angelica.transform.compat.extrautils;
+
+import com.gtnewhorizons.angelica.AngelicaMod;
+import com.gtnewhorizons.angelica.loading.AngelicaTweaker;
+import net.minecraft.launchwrapper.IClassTransformer;
+import org.jetbrains.annotations.NotNull;
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.tree.AnnotationNode;
+import org.objectweb.asm.tree.ClassNode;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class RenderBlockColorTransformer implements IClassTransformer {
+
+    @Override
+    public byte[] transform(String name, String transformedName, byte[] basicClass) {
```

Comment:
NIT: Might be better to do one transformer with a map of the classes and perThread true/false for each?

Reply:
Also might be worth using the class constant parser to ensure this doesn't blindly try to transform every class and slow startup down.

Reply:
I believe this is handled with the latest commit, it's not using class constant parser but I think achieves the same goal